<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>Institute for Machine Learning @ JKU | AI in Drug Discovery</title>
<meta name="description" content="Research blog of the Institute for Machine Learning @ JKU.
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

<!-- Styles -->
<link rel="shortcut icon" href="/assets/img/favicon.ico">
<link rel="stylesheet" href="/assets/css/main.css">

<link rel="canonical" href="/research/ai_drug/">

<!-- Theming-->



  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-KC0VGMCD6W"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'G-KC0VGMCD6W');
  </script>


    
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
      <a class="navbar-brand title font-weight-lighter" href="ml-jku.github.io/">
       <!-- <span>Institute</span> for Machine  Learning @ JKU -->
       <img class="rounded" src="/assets/img/jku_ml_1.svg" width=100px height=45px>
       <img class="rounded" src="/assets/img/ellis_linz.jpeg" width=200px height=45px>
      </a>
    <div class="container">
      
      <!-- Navbar Toogle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/">
              about
              
            </a>
          </li>
          
          <!-- Blog -->
          <li class="nav-item ">
            <a class="nav-link" href="/blog/">
              blog
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          <li class="nav-item ">
                
                    <div class="dropdown">
                      <a class="nav-link" href="/research/">
                        research
                        
                      </a>
                        <div class="dropdown-content">
                          <a href="/research/deep_learning">Deep Learning</a>
                          <a href="/research/ai_healthcare">AI in Health Care</a>
                          <a href="/research/ai_drug">AI in Drug Discovery</a>
                          <a href="/research/reinforcement_learning">Reinforcement Learning</a>
                          <a href="/research/hopfield_networks">Modern Hopfield Networks</a>
                          <a href="/research/ai4earth">AI 4 Earth</a>
                          <a href="/research/ai4drive">AI 4 Driving</a>
                        </div>
                      </div>
                    
                
          </li>
          
          
          
          
          
          
          
          <li class="nav-item ">
                
                <a class="nav-link" href="/publications/">
                  publications
                  
                </a>
                
          </li>
          
          
          
          
          
          
          
          
          
          
          
          <li class="nav-item ">
                
                <a class="nav-link" href="/talks/">
                  talks
                  
                </a>
                
          </li>
          
          
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="https://www.jku.at/en/institute-for-machine-learning/about-us/team/" target="_blank" rel="noopener noreferrer">
                people
                
              </a>
          </li>
          
          
          
          
          
          <li class="nav-item ">
                
                <a class="nav-link" href="/software/">
                  software
                  
                </a>
                
          </li>
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="https://www.jku.at/en/institute-for-machine-learning/" target="_blank" rel="noopener noreferrer">
                contact
                
              </a>
          </li>
          
          
          
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    
    <h1 class="post-title">AI in Drug Discovery</h1>
    <p class="post-description"></p>
    
  </header>

  <article>
    <div class="about-text">
    <p>Artificial Intelligence (AI) methods have been shown to be able to design new molecules and to accurately foresee their role in the human body. New drugs will be safer and more effective than they have ever been. Researchers of JKU have won the Tox21 Data Challenge demonstrating that neural AIs can find toxic effects. <a href="https://jku.at/en/institute-for-machine-learning/about-us/team/assist-prof-mag-dr-guenter-klambauer/">Günter Klambauer</a> leads an <a href="https://ncats.nih.gov/news/releases/2015/tox21-challenge-2014-winners">award-winning</a> team of scientists developing AIs that design, improve and assess new drugs. Several AI-generated drugs are currently tested for their ability to inhibit SARS-CoV-2.</p>

<div class="row">
      <div class="col-sm mt-3 mt-md-0">
          <img class="img-fluid rounded" src="/assets/img/research/ai_drug/ai_drug_1.png" alt="" title="example image" />
      </div>
  </div>

<div class="publications">
  <h4>recent publications in AI in Drug Discovery:</h4>
    <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">AI4Science</abbr>
    
  
  </div>

  <div id="svensson2022robust" class="col-sm-8">
    
      <div class="title">Robust task-specific adaption of models for drug-target interaction prediction</div>
      <div class="author">
        
          
            
              
                
                  Svensson, E.,
                
              
            
          
        
          
            
              
                
                  Hoedt, P.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>In NeurIPS 2022 AI for Science: Progress and Promises</em>
      
      
        2022
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://openreview.net/pdf?id=dIX34JWnIAL" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
      <a href="https://github.com/ml-jku/hyper-dti" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>HyperNetworks have been established as an effective technique to achieve fast adaptation of parameters for neural networks. Recently, HyperNetworks conditioned on descriptors of tasks have improved multi-task generalization in various domains, such as personalized federated learning and neural architecture search. Especially powerful results were achieved in few- and zero-shot settings, attributed to the increased information sharing by the HyperNetwork. With the rise of new diseases fast discovery of drugs is needed which requires proteo-chemometric models that are able to generalize drug-target interaction predictions in low-data scenarios. State-of-the-art methods apply a few fully-connected layers to concatenated learned embeddings of the protein target and drug compound. In this work, we develop a task-conditioned HyperNetwork approach for the problem of predicting drug-target interactions in drug discovery. We show that when model parameters are predicted for the fully-connected layers processing the drug compound embedding, based on the protein target embedding, predictive performance can be improved over previous methods. Two additional components of our architecture, a) switching to L1 loss, and b) integrating a context module for proteins, further boost performance and robustness. On an established benchmark for proteo-chemometrics models, our architecture outperforms previous methods in all settings, including few- and zero-shot settings. In an ablation study, we analyze the importance of each of the components of our HyperNetwork approach.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">ML4Molecules</abbr>
    
  
  </div>

  <div id="svensson2022task" class="col-sm-8">
    
      <div class="title">Task-conditioned modeling of drug-target interactions</div>
      <div class="author">
        
          
            
              
                
                  Svensson, E.,
                
              
            
          
        
          
            
              
                
                  Hoedt, P.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>In ELLIS Machine Learning for Molecule Discovery Workshop</em>
      
      
        2022
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://openreview.net/pdf?id=MrUwwGKRhOM" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
      <a href="https://github.com/ml-jku/hyper-dti" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>HyperNetworks have been established as an effective technique to achieve fast adaptation of parameters for neural networks. Recently, HyperNetworks conditioned on descriptors of tasks have improved multi-task generalization in various domains, such as personalized federated learning and neural architecture search. Especially powerful results were achieved in few- and zero-shot settings, attributed to the increased information sharing by the HyperNetwork. With the rise of new diseases fast discovery of drugs is needed which requires proteo-chemometric models that are able to generalize drug-target interaction predictions in low-data scenarios. State-of-the-art methods apply a few fully-connected layers to concatenated learned embeddings of the protein target and drug compound. In this work, we develop a task-conditioned HyperNetwork approach for the problem of predicting drug-target interactions in drug discovery. We show that when model parameters are predicted for the fully-connected layers processing the drug compound embedding, based on the protein target embedding, predictive performance can be improved over previous methods. Two additional components of our architecture, a) switching to L1 loss, and b) integrating a context module for proteins, further boost performance and robustness. On an established benchmark for proteo-chemometrics models, our architecture outperforms previous methods in all settings, including few- and zero-shot settings. In an ablation study, we analyze the importance of each of the components of our HyperNetwork approach.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">JCIM</abbr>
    
  
  </div>

  <div id="seidlImproving2022" class="col-sm-8">
    
      <div class="title">Improving Few- and Zero-Shot Reaction Template Prediction Using Modern Hopfield Networks</div>
      <div class="author">
        
          
            
              
                
                  Seidl, P.,
                
              
            
          
        
          
            
              
                
                  Renz, P.,
                
              
            
          
        
          
            
              
                
                  Dyubankova, N.,
                
              
            
          
        
          
            
              
                
                  Neves, P.,
                
              
            
          
        
          
            
              
                
                  Verhoeven, J.,
                
              
            
          
        
          
            
              
                
                  Wegner, J.,
                
              
            
          
        
          
            
              
                
                  Segler, M.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Journal of Chemical Information and Modeling</em>
      
      
        2022
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1021/acs.jcim.1c01065" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
      <a href="https://github.com/ml-jku/mhn-react" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Finding synthesis routes for molecules of interest is essential in the discovery of new drugs and materials. To find such routes, computer-assisted synthesis planning (CASP) methods are employed, which rely on a single-step model of chemical reactivity. In this study, we introduce a template-based single-step retrosynthesis model based on Modern Hopfield Networks, which learn an encoding of both molecules and reaction templates in order to predict the relevance of templates for a given molecule. The template representation allows generalization across different reactions and significantly improves the performance of template relevance prediction, especially for templates with few or zero training examples. With inference speed up to orders of magnitude faster than baseline methods, we improve or match the state-of-the-art performance for top-k exact match accuracy for k ≥ 3 in the retrosynthesis benchmark USPTO-50k. Code to reproduce the results is available at github.com/ml-jku/mhn-react.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">QSAR</abbr>
    
  
  </div>

  <div id="seidlBenchmarking2021" class="col-sm-8">
    
      <div class="title">Benchmarking recent Deep Learning methods on the extended Tox21 data set</div>
      <div class="author">
        
          
            
              
                
                  Seidl, P.,
                
              
            
          
        
          
            
              
                
                  Halmich, C.,
                
              
            
          
        
          
            
              
                
                  Mayr, A.,
                
              
            
          
        
          
            
              
                
                  Vall, A.,
                
              
            
          
        
          
            
              
                
                  Ruch, P.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>In </em>
      
      
        2021
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.ascctox.org/sites/default/files/QSAR%202021%20Program.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The Tox21 data set has evolved into a standard benchmark for computational QSAR methods in toxicology. One limitation of the Tox21 data set is, however, that it only contains twelve toxic assays which strongly restricts its power to distinguish the strength of computational methods. We ameliorate this problem by benchmarking on the extended Tox21 dataset with 68 publicly available assays in order to allow for a better assessment and characterization. The broader range of assays also allows for multi-task approaches, which have been particularly successful as predictive models. Furthermore, previous publications comparing methods on Tox21 did not include recent developments in the field of machine learning, such as graph neural and modern Hopfield networks. Thus we benchmark a set of prominent machine learning methods including those new types of neural networks. The results of the benchmarking study show that the best methods are modern Hopfield networks and multi-task graph neural networks with an average area-under-ROCcurve of 0.91 ± 0.05 (standard deviation across assays), while traditional methods, such as Random Forests fall behind by a substantial margin. Our results of the full benchmark suggest that multi-task learning has a stronger effect on the predictive performance than the choice of the representation of the molecules, such as graph, descriptors, or fingerprints.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Frontiers in AI</abbr>
    
  
  </div>

  <div id="vall_promise_2021" class="col-sm-8">
    
      <div class="title">The Promise of AI for DILI Prediction</div>
      <div class="author">
        
          
            
              
                
                  Vall, A.,
                
              
            
          
        
          
            
              
                
                  Sabnis, Y.,
                
              
            
          
        
          
            
              
                
                  Shi, J.,
                
              
            
          
        
          
            
              
                
                  Class, R.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Frontiers in Artificial Intelligence</em>
      
      
        2021
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.frontiersin.org/articles/10.3389/frai.2021.638410" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Drug-induced liver injury (DILI) is a common reason for the withdrawal of a drug from the market. Early assessment of DILI risk is an essential part of drug development, but it is rendered challenging prior to clinical trials by the complex factors that give rise to liver damage. Artificial intelligence (AI) approaches, particularly those building on machine learning, range from random forests to more recent techniques such as deep learning, and provide tools that can analyze chemical compounds and accurately predict some of their properties based purely on their structure. This article reviews existing AI approaches to predicting DILI and elaborates on the challenges that arise from the as yet limited availability of data. Future directions are discussed focusing on rich data modalities, such as 3D spheroids, and the slow but steady increase in drugs annotated with DILI risk labels.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">arXiv</abbr>
    
  
  </div>

  <div id="seidl2021reaction" class="col-sm-8">
    
      <div class="title">Modern Hopfield Networks for Few- and Zero-Shot Reaction Prediction</div>
      <div class="author">
        
      </div>

      <div class="periodical">
      
        <em>arXiv preprint arXiv:2104.03279</em>
      
      
        2021
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>An essential step in the discovery of new drugs and materials is the synthesis of a molecule that exists so far only as an idea to test its biological and physical properties. While computer-aided design of virtual molecules has made large progress, computer-assisted synthesis planning (CASP) to realize physical molecules is still in its infancy and lacks a performance level that would enable large-scale molecule discovery. CASP supports the search for multi-step synthesis routes, which is very challenging due to high branching factors in each synthesis step and the hidden rules that govern the reactions. The central and repeatedly applied step in CASP is reaction prediction, for which machine learning methods yield the best performance. We propose a novel reaction prediction approach that uses a deep learning architecture with modern Hopfield networks (MHNs) that is optimized by contrastive learning. An MHN is an associative memory that can store and retrieve chemical reactions in each layer of a deep learning architecture. We show that our MHN contrastive learning approach enables few- and zero-shot learning for reaction prediction which, in contrast to previous methods, can deal with rare, single, or even no training example(s) for a reaction. On a well established benchmark, our MHN approach pushes the state-of-the-art performance up by a large margin as it improves the predictive top-100 accuracy from 0.858±0.004 to 0.959±0.004. This advance might pave the way to large-scale molecule discovery.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">QSAR2021</abbr>
    
  
  </div>

  <div id="schimunekhERGinterpretability2021" class="col-sm-8">
    
      <div class="title">Comparative assessment of interpretability methods of deep activity models for hERG</div>
      <div class="author">
        
          
            
              
                
                  Schimunek, J.,
                
              
            
          
        
          
            
              
                
                  Friedrich, L.,
                
              
            
          
        
          
            
              
                
                  Kuhn, D.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  Rippmann, F.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
      
        2021
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.ascctox.org/qsar2021" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Since many highly accurate predictive models for bioactivity and toxicity assays are based on Deep Learning methods, there has been a recent surge of interest in interpretability methods for Deep Learning approaches in drug discovery [1,2]. Interpretability methods are highly desired by human experts to enable them to make design decisions on the molecule based on the activity model. However, it is still unclear which of those interpretability methods are better identifying relevant substructures of molecules.  A method comparison is further complicated by the lack of ground truth and appropriate metrics. Here, we present the first comparative study of a set of interpretability methods for Deep Learning models for hERG inhibition. In our work, we compared layer-wise relevance propagation, feature gradients, saliency maps, integrated gradients, occlusion and Shapley values. In the quantitative analysis, known substructures which indicate hERG activity are used as ground truth [3]. Interpretability methods were compared by their ability to rank atoms, which are part of indicative substructures, first. The significantly best performing method is Shapley values with an area under-ROC-curve (AUC) of  0.74 ± 0.12, but also runner-up methods, such as Integrated Gradients, achieved similar results. The results indicate that interpretability methods for deep activity models have the potential to identify new toxicophores.

[1] Jiménez-Luna, J., et al. (2020). Nature Machine Intelligence, 2(10), 573-584.
[2]  Preuer, K., et al. (2019). In Explainable AI: Interpreting, Explaining and Visualizing Deep Learning (pp. 331-345).
[3] Czodrowski, P. (2013). Journal of chemical information and modeling, 53, 2240–2251.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Springer</abbr>
    
  
  </div>

  <div id="sturmIndustryscaleApplicationEvaluation2020" class="col-sm-8">
    
      <div class="title">Industry-Scale Application and Evaluation of Deep Learning for Drug Target Prediction</div>
      <div class="author">
        
          
            
              
                
                  Sturm, N.,
                
              
            
          
        
          
            
              
                
                  Mayr, A.,
                
              
            
          
        
          
            
              
                
                  Le Van, T.,
                
              
            
          
        
          
            
              
                
                  Chupakhin, V.,
                
              
            
          
        
          
            
              
                
                  Ceulemans, H.,
                
              
            
          
        
          
            
              
                
                  Wegner, J.,
                
              
            
          
        
          
            
              
                
                  Golib-Dzib, J.,
                
              
            
          
        
          
            
              
                
                  Jeliazkova, N.,
                
              
            
          
        
          
            
              
                
                  Vandriessche, Y.,
                
              
            
          
        
          
            
              
                
                  Böhm, S.,
                
              
            
          
        
          
            
              
                
                  Cima, V.,
                
              
            
          
        
          
            
              
                
                  Martinovic, J.,
                
              
            
          
        
          
            
              
                
                  Greene, N.,
                
              
            
          
        
          
            
              
                
                  Vander Aa, T.,
                
              
            
          
        
          
            
              
                
                  Ashby, T.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  Engkvist, O.,
                
              
            
          
        
          
            
              
                
                  Klambauer, G.,
                
              
            
          
        
          
            
              
                
                  and Chen, H.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2020
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1186/s13321-020-00428-5" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Artificial intelligence (AI) is undergoing a revolution thanks to the breakthroughs of machine learning algorithms in computer vision, speech recognition, natural language processing and generative modelling. Recent works on publicly available pharmaceutical data showed that AI methods are highly promising for Drug Target prediction. However, the quality of public data might be different than that of industry data due to different labs reporting measurements, different measurement techniques, fewer samples and less diverse and specialized assays. As part of a European funded project (ExCAPE), that brought together expertise from pharmaceutical industry, machine learning, and high-performance computing, we investigated how well machine learning models obtained from public data can be transferred to internal pharmaceutical industry data. Our results show that machine learning models trained on public data can indeed maintain their predictive power to a large degree when applied to industry data. Moreover, we observed that deep learning derived machine learning models outperformed comparable models, which were trained by other machine learning algorithms, when applied to internal pharmaceutical company datasets. To our knowledge, this is the first large-scale study evaluating the potential of machine learning and especially deep learning directly at the level of industry-scale settings and moreover investigating the transferability of publicly learned target prediction models towards industrial bioactivity prediction pipelines.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="renzFailureModesMolecule2020a" class="col-sm-8">
    
      <div class="title">On Failure Modes in Molecule Generation and Optimization</div>
      <div class="author">
        
          
            
              
                
                  Renz, P.,
                
              
            
          
        
          
            
              
                
                  Van Rompaey, D.,
                
              
            
          
        
          
            
              
                
                  Wegner, J.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2020
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1016/j.ddtec.2020.09.003" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
      <a href="https://github.com/ml-jku/mgenerators-failure-modes" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>There has been a wave of generative models for molecules triggered by advances in the field of Deep Learning. These generative models are often used to optimize chemical compounds towards particular properties or a desired biological activity. The evaluation of generative models remains challenging and suggested performance metrics or scoring functions often do not cover all relevant aspects of drug design projects. In this work, we highlight some unintended failure modes in molecular generation and optimization and how these evade detection by current performance metrics.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="mendenCommunityAssessmentAdvance2019" class="col-sm-8">
    
      <div class="title">Community Assessment to Advance Computational Prediction of Cancer Drug Combinations in a Pharmacogenomic Screen</div>
      <div class="author">
        
          
            
              
                
                  Menden, M.,
                
              
            
          
        
          
            
              
                
                  Wang, D.,
                
              
            
          
        
          
            
              
                
                  Mason, M.,
                
              
            
          
        
          
            
              
                
                  Szalai, B.,
                
              
            
          
        
          
            
              
                
                  Bulusu, K.,
                
              
            
          
        
          
            
              
                
                  Guan, Y.,
                
              
            
          
        
          
            
              
                
                  Yu, T.,
                
              
            
          
        
          
            
              
                
                  Kang, J.,
                
              
            
          
        
          
            
              
                
                  Jeon, M.,
                
              
            
          
        
          
            
              
                
                  Wolfinger, R.,
                
              
            
          
        
          
            
              
                
                  Nguyen, T.,
                
              
            
          
        
          
            
              
                
                  Zaslavskiy, M.,
                
              
            
          
        
          
            
              
                
                  Jang, I.,
                
              
            
          
        
          
            
              
                
                  Ghazoui, Z.,
                
              
            
          
        
          
            
              
                
                  Ahsen, M.,
                
              
            
          
        
          
            
              
                
                  Vogel, R.,
                
              
            
          
        
          
            
              
                
                  Neto, E.,
                
              
            
          
        
          
            
              
                
                  Norman, T.,
                
              
            
          
        
          
            
              
                
                  Tang, E.,
                
              
            
          
        
          
            
              
                
                  Garnett, M.,
                
              
            
          
        
          
            
              
                
                  Veroli, G.,
                
              
            
          
        
          
            
              
                
                  Fawell, S.,
                
              
            
          
        
          
            
              
                
                  Stolovitzky, G.,
                
              
            
          
        
          
            
              
                
                  Guinney, J.,
                
              
            
          
        
          
            
              
                
                  Dry, J.,
                
              
            
          
        
          
            
              
                
                  and Saez-Rodriguez, J.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2019
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1038/s41467-019-09799-2" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The effectiveness of most cancer targeted therapies is short-lived. Tumors often develop resistance that might be overcome with drug combinations. However, the number of possible combinations is vast, necessitating data-driven approaches to find optimal patient-specific treatments. Here we report AstraZeneca’s large drug combination dataset, consisting of 11,576 experiments from 910 combinations across 85 molecularly characterized cancer cell lines, and results of a DREAM Challenge to evaluate computational strategies for predicting synergistic drug pairs and biomarkers. 160 teams participated to provide a comprehensive methodological development and benchmarking. Winning methods incorporate prior knowledge of drug-target interactions. Synergy is predicted with an accuracy matching biological replicates for &gt;60% of combinations. However, 20% of drug combinations are poorly predicted by all methods. Genomic rationale for synergy predictions are identified, including ADAM17 inhibitor antagonism when combined with PIK3CB/D inhibition contrasting to synergy when combined with other PI3K-pathway inhibitors in PIK3CA mutant cells.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="klambauerMachineLearningDrug2019" class="col-sm-8">
    
      <div class="title">Machine Learning in Drug Discovery</div>
      <div class="author">
        
          
            
              
                
                  Klambauer, G.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Rarey, M.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2019
      
      </div>
    

    <div class="links">
    
    
    
      <a href="https://doi.org/10.1021/acs.jcim.9b00136" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="preuerInterpretableDeepLearning2019" class="col-sm-8">
    
      <div class="title">Interpretable Deep Learning in Drug Discovery</div>
      <div class="author">
        
          
            
              
                
                  Preuer, K.,
                
              
            
          
        
          
            
              
                
                  Klambauer, G.,
                
              
            
          
        
          
            
              
                
                  Rippmann, F.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Unterthiner, T.
                
              
            
          
        
      </div>

      <div class="periodical">
      
      
        2019
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1007/978-3-030-28954-6_18" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Without any means of interpretation, neural networks that predict molecular properties and bioactivities are merely black boxes. We will unravel these black boxes and will demonstrate approaches to understand the learned representations which are hidden inside these models. We show how single neurons can be interpreted as classifiers which determine the presence or absence of pharmacophore- or toxicophore-like structures, thereby generating new insights and relevant knowledge for chemistry, pharmacology and biochemistry. We further discuss how these novel pharmacophores/toxicophores can be determined from the network by identifying the most relevant components of a compound for the prediction of the network. Additionally, we propose a method which can be used to extract new pharmacophores from a model and will show that these extracted structures are consistent with literature findings. We envision that having access to such interpretable knowledge is a crucial aid in the development and design of new pharmaceutically active molecules, and helps to investigate and understand failures and successes of current methods.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">NeurIPS</abbr>
    
  
  </div>

  <div id="renzUncertaintyEstimationMethods2019" class="col-sm-8">
    
      <div class="title">Uncertainty Estimation Methods to Support Decision-Making in Early Phases of Drug Discovery</div>
      <div class="author">
        
          
            
              
                
                  Renz, P.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2019
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>It takes about a decade to develop a new drug by a process in which a large number of decisions have to be made. Those decisions are critical for the success or failure of a multi-million dollar drug discovery project, which could save many lives or increase life quality. Decisions in early phases of drug discovery, such as the selection of certain series of chemical compounds, are particularly impactful on the success rate. Machine learning models are increasingly used to inform the decision making process by predicting desired effects, undesired effects, such as toxicity, molecular properties, or which wet-lab test to perform next. Thus, accurately quantifying the uncertainties of the models’ outputs is critical, for example, in order to calculate expected utilities, to estimate the risk and the potential gain. In this work, we review, assess and compare recent uncertainty estimation methods with respect to their use in drug discovery projects. We test both, which methods give well calibrated prediction and which ones perform well at misclassification detection. For the latter, we find the entropy of the predictive distribution performs best. Finally, we discuss the problem of defining out-of-distribution samples for prediction tasks on chemical compounds.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="hochreiterMachineLearningDrug2018" class="col-sm-8">
    
      <div class="title">Machine Learning in Drug Discovery</div>
      <div class="author">
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  Klambauer, G.,
                
              
            
          
        
          
            
              
                
                  and Rarey, M.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
    
    
      <a href="https://doi.org/10.1021/acs.jcim.8b00478" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="mayrLargescaleComparisonMachine2018" class="col-sm-8">
    
      <div class="title">Large-Scale Comparison of Machine Learning Methods for Drug Target Prediction on ChEMBL</div>
      <div class="author">
        
          
            
              
                
                  Mayr, A.,
                
              
            
          
        
          
            
              
                
                  Klambauer, G.,
                
              
            
          
        
          
            
              
                
                  Unterthiner, T.,
                
              
            
          
        
          
            
              
                
                  Steijaert, M.,
                
              
            
          
        
          
            
              
                
                  Wegner, J.,
                
              
            
          
        
          
            
              
                
                  Ceulemans, H.,
                
              
            
          
        
          
            
              
                
                  Clevert, D.,
                
              
            
          
        
          
            
              
                
                  and Hochreiter, S.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1039/C8SC00148K" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Deep learning is currently the most successful machine learning technique in a wide range of application areas and has recently been applied successfully in drug discovery research to predict potential drug targets and to screen for active molecules. However, due to (1) the lack of large-scale studies, (2) the compound series bias that is characteristic of drug discovery datasets and (3) the hyperparameter selection bias that comes with the high number of potential deep learning architectures, it remains unclear whether deep learning can indeed outperform existing computational methods in drug discovery tasks. We therefore assessed the performance of several deep learning methods on a large-scale drug discovery dataset and compared the results with those of other machine learning and target prediction methods. To avoid potential biases from hyperparameter selection or compound series, we used a nested cluster-cross-validation strategy. We found (1) that deep learning methods significantly outperform all competing methods and (2) that the predictive performance of deep learning is in many cases comparable to that of tests performed in wet labs (i.e., in vitro assays).</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="preuerDeepSynergyPredictingAnticancer2018" class="col-sm-8">
    
      <div class="title">DeepSynergy: Predicting Anti-Cancer Drug Synergy with Deep Learning</div>
      <div class="author">
        
          
            
              
                
                  Preuer, K.,
                
              
            
          
        
          
            
              
                
                  Lewis, R.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  Bender, A.,
                
              
            
          
        
          
            
              
                
                  Bulusu, K.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1093/bioinformatics/btx806" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>While drug combination therapies are a well-established concept in cancer treatment, identifying novel synergistic combinations is challenging due to the size of combinatorial space. However, computational approaches have emerged as a time- and cost-efficient way to prioritize combinations to test, based on recently available large-scale combination screening data. Recently, Deep Learning has had an impact in many research areas by achieving new state-of-the-art model performance. However, Deep Learning has not yet been applied to drug synergy prediction, which is the approach we present here, termed DeepSynergy. DeepSynergy uses chemical and genomic information as input information, a normalization strategy to account for input data heterogeneity, and conical layers to model drug synergies.DeepSynergy was compared to other machine learning methods such as Gradient Boosting Machines, Random Forests, Support Vector Machines and Elastic Nets on the largest publicly available synergy dataset with respect to mean squared error. DeepSynergy significantly outperformed the other methods with an improvement of 7.2% over the second best method at the prediction of novel drug combinations within the space of explored drugs and cell lines. At this task, the mean Pearson correlation coefficient between the measured and the predicted values of DeepSynergy was 0.73. Applying DeepSynergy for classification of these novel drug combinations resulted in a high predictive performance of an AUC of 0.90. Furthermore, we found that all compared methods exhibit low predictive performance when extrapolating to unexplored drugs or cell lines, which we suggest is due to limitations in the size and diversity of the dataset. We envision that DeepSynergy could be a valuable tool for selecting novel synergistic drug combinations.DeepSynergy is available via www.bioinf.jku.at/software/DeepSynergy.Supplementary data are available at Bioinformatics online.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="preuerFrechetChemNetDistance2018" class="col-sm-8">
    
      <div class="title">Fréchet ChemNet Distance: A Metric for Generative Models for Molecules in Drug Discovery</div>
      <div class="author">
        
          
            
              
                
                  Preuer, K.,
                
              
            
          
        
          
            
              
                
                  Renz, P.,
                
              
            
          
        
          
            
              
                
                  Unterthiner, T.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Klambauer, G.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1021/acs.jcim.8b00234" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The new wave of successful generative models in machine learning has increased the interest in deep learning driven de novo drug design. However, method comparison is difficult because of various flaws of the currently employed evaluation metrics. We propose an evaluation metric for generative models called Fréchet ChemNet distance (FCD). The advantage of the FCD over previous metrics is that it can detect whether generated molecules are diverse and have similar chemical and biological properties as real molecules.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="clevertRectifiedFactorNetworks2017" class="col-sm-8">
    
      <div class="title">Rectified Factor Networks for Biclustering of Omics Data</div>
      <div class="author">
        
          
            
              
                
                  Clevert, D.,
                
              
            
          
        
          
            
              
                
                  Unterthiner, T.,
                
              
            
          
        
          
            
              
                
                  Povysil, G.,
                
              
            
          
        
          
            
              
                
                  and Hochreiter, S.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2017
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1093/bioinformatics/btx226" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Biclustering has become a major tool for analyzing large datasets given as matrix of samples times features and has been successfully applied in life sciences and e-commerce for drug design and recommender systems, respectively. Factor Analysis for Bicluster Acquisition (FABIA), one of the most successful biclustering methods, is a generative model that represents each bicluster by two sparse membership vectors: one for the samples and one for the features. However, FABIA is restricted to about 20 code units because of the high computational complexity of computing the posterior. Furthermore, code units are sometimes insufficiently decorrelated and sample membership is difficult to determine. We propose to use the recently introduced unsupervised Deep Learning approach Rectified Factor Networks (RFNs) to overcome the drawbacks of existing biclustering methods. RFNs efficiently construct very sparse, non-linear, high-dimensional representations of the input via their posterior means. RFN learning is a generalized alternating minimization algorithm based on the posterior regularization method which enforces non-negative and normalized posterior means. Each code unit represents a bicluster, where samples for which the code unit is active belong to the bicluster and features that have activating weights to the code unit belong to the bicluster.On 400 benchmark datasets and on three gene expression datasets with known clusters, RFN outperformed 13 other biclustering methods including FABIA. On data of the 1000 Genomes Project, RFN could identify DNA segments which indicate, that interbreeding with other hominins starting already before ancestors of modern humans left Africa.https://github.com/bioinf-jku/librfn</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="povysilPanelcnMOPSCopynumber2017" class="col-sm-8">
    
      <div class="title">Panelcn.MOPS: Copy-Number Detection in Targeted NGS Panel Data for Clinical Diagnostics</div>
      <div class="author">
        
          
            
              
                
                  Povysil, G.,
                
              
            
          
        
          
            
              
                
                  Tzika, A.,
                
              
            
          
        
          
            
              
                
                  Vogt, J.,
                
              
            
          
        
          
            
              
                
                  Haunschmid, V.,
                
              
            
          
        
          
            
              
                
                  Messiaen, L.,
                
              
            
          
        
          
            
              
                
                  Zschocke, J.,
                
              
            
          
        
          
            
              
                
                  Klambauer, G.,
                
              
            
          
        
          
            
              
                
                  Hochreiter, S.,
                
              
            
          
        
          
            
              
                
                  and Wimmer, K.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em></em>
      
      
        2017
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1002/humu.23237" class="btn btn-sm z-depth-0" role="button" target="_blank">url</a>
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Targeted next-generation-sequencing (NGS) panels have largely replaced Sanger sequencing in clinical diagnostics. They allow for the detection of copy-number variations (CNVs) in addition to single-nucleotide variants and small insertions/deletions. However, existing computational CNV detection methods have shortcomings regarding accuracy, quality control (QC), incidental findings, and user-friendliness. We developed panelcn.MOPS, a novel pipeline for detecting CNVs in targeted NGS panel data. Using data from 180 samples, we compared panelcn.MOPS with five state-of-the-art methods. With panelcn.MOPS leading the field, most methods achieved comparably high accuracy. panelcn.MOPS reliably detected CNVs ranging in size from part of a region of interest (ROI), to whole genes, which may comprise all ROIs investigated in a given sample. The latter is enabled by analyzing reads from all ROIs of the panel, but presenting results exclusively for user-selected genes, thus avoiding incidental findings. Additionally, panelcn.MOPS offers QC criteria not only for samples, but also for individual ROIs within a sample, which increases the confidence in called CNVs. panelcn.MOPS is freely available both as R package and standalone software with graphical user interface that is easy to use for clinical geneticists without any programming experience. panelcn.MOPS combines high sensitivity and specificity with user-friendliness rendering it highly suitable for routine clinical diagnostics.</p>
    </div>
    
  </div>
</div>
</li></ol>
  </div>

  </div>
  </article>

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    &copy; Copyright 2025 Institute for Machine Learning @ JKU.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>.

    
  </div>
</footer>



  </body>

  <!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>

<!-- Load DarkMode JS -->
<script src="/assets/js/dark_mode.js"></script>


</html>
